processing/src/main/java/org/apache/druid/query/filter/BoundDimFilter.java

```diff
@@ -152,6 +171,12 @@ public StringComparator getOrdering()
     return ordering;
   }
 
+  @JsonProperty
```

Comment:
Please include `@JsonInclude(JsonInclude.Include.NON_NULL)` here, and in all other places FilterTuning appears as a json property, to avoid polluting jsonifications of filters with this experimental feature.

Reply:
added to `DimFilter` getters